worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -3084,4 +3088,33 @@ public void run(Object value) {
         }
         return false;
     }
+
+    @Override
+    public List<? extends Entity> getEntities() {
+        return world.getEntities();
+    }
+    
+    @Override
+    public List<? extends Entity> getEntities(Region region) {
+        return world.getEntities(region);
+    }
+    
+    @Override
+    public Entity createEntity(Location location, BaseEntity entity){
+        try {
+            return this.getExtent().createEntity(location, entity);
+        } catch (WorldEditException e) {
+            throw new RuntimeException("Unexpected exception", e);
+        }
+    }
+
+    @Override
+    public void removeEntity(int x, int y, int z, UUID uuid)
+    {
```

Comment:
ew

Reply:
what's wrong with that?

Reply:
Bracket should be on the same line as the method.

Reply:
fixed in https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/518/commits/947871281f87c0001239cfe78a642322f2361a8b